infra/feast-operator/README.md

```diff
@@ -0,0 +1,118 @@
+# feast-operator
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+
+### Prerequisites
+- go version v1.21.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+### To Deploy on the cluster
+**Build and push your image to the location specified by `IMG`:**
+
+```sh
+make docker-build docker-push IMG=<some-registry>/feast-operator:tag
```

Comment:
We should update this as well.

Reply:
this has been moved to the contribution section of the README and i've improved this "deploy" section.